examples/platform/linux/AppMain.cpp

```diff
@@ -279,10 +280,20 @@ int ChipLinuxAppInit(int argc, char * const argv[], OptionSet * customOptions)
         const char * traceFilename = LinuxDeviceOptions::GetInstance().traceStreamFilename.Value().c_str();
         gTraceStream               = new chip::trace::TraceStreamFile(traceFilename);
     }
-    else if (LinuxDeviceOptions::GetInstance().traceStreamToLogEnabled)
+    else if (LinuxDeviceOptions::GetInstance().traceStreamToLog == 1)
     {
         gTraceStream = new chip::trace::TraceStreamLog();
     }
+    else if (LinuxDeviceOptions::GetInstance().traceStreamToLog == 2)
```

Comment:
We need to have *mutually exclusive* arguments, so we can have both the `chip::trace::TraceStreamLog` and your new decoder separately, rather than a tri-state 0/1/2 --log_stream

Reply:
I have added a `trace_decode` option. I have verified that both can be used in parallels.